fuel-indexer/src/executor.rs

```diff
@@ -228,166 +228,169 @@ impl Executor for WasmIndexExecutor {
-        match executor {
-            Err(IndexerError::MissingHandler) => (),
-            e => panic!("Expected missing handler error {:#?}", e),
-        }
-
-        let executor = WasmIndexExecutor::new(database_url.to_string(), manifest, WASM_BYTES).await;
-        assert!(executor.is_ok());
-
-        let mut executor = executor.unwrap();
-
-        let evt1 = SomeEvent {
-            id: 1020,
-            account: [0xaf; 32],
-        };
-        let evt2 = AnotherEvent {
-            id: 100,
-            account: [0x5a; 32],
-            hash: [0x43; 32],
-        };
-
-        let some_event = ABIEncoder::new()
-            .encode(&[evt1.into_token()])
-            .expect("Failed to encode");
-        let another_event = ABIEncoder::new()
-            .encode(&[evt2.into_token()])
-            .expect("Failed to encode");
-
-        let result = executor
-            .handle_events(vec![BlockData {
-                height: 0,
-                transactions: vec![
-                    vec![
-                        Receipt::Call {
-                            id: [0u8; 32].into(),
-                            to: [0u8; 32].into(),
-                            amount: 400,
-                            asset_id: [0u8; 32].into(),
-                            gas: 4,
-                            param1: 2048508220,
-                            param2: 0,
-                            pc: 0,
-                            is: 0,
-                        },
-                        Receipt::ReturnData {
-                            id: [0u8; 32].into(),
-                            ptr: 2342143,
-                            len: some_event.len() as u64,
-                            digest: [0u8; 32].into(),
-                            data: some_event,
-                            pc: 0,
-                            is: 0,
-                        },
-                    ],
-                    vec![
-                        Receipt::Call {
-                            id: [0u8; 32].into(),
-                            to: [0u8; 32].into(),
-                            amount: 400,
-                            asset_id: [0u8; 32].into(),
-                            gas: 4,
-                            param1: 2379805026,
-                            param2: 0,
-                            pc: 0,
-                            is: 0,
-                        },
-                        Receipt::ReturnData {
-                            id: [0u8; 32].into(),
-                            ptr: 2342143,
-                            len: another_event.len() as u64,
-                            digest: [0u8; 32].into(),
-                            data: another_event,
-                            pc: 0,
-                            is: 0,
-                        },
-                    ],
-                ],
-            }])
-            .await;
-        assert!(result.is_ok());
-    }
-}
+// #[cfg(test)]
```

Comment:
@tjsharp1 
- Obviously would prefer to leave these tests in as they're valuable (and we already don't have that much testing)
- But absent a good resolution, can't let them block this work for too much longer. 
- I can create a high priority tracking issue to make sure we don't forget about this, but yea -- I think I've reached the end of my debugging ideas with this 🤷